jabgui/src/main/java/org/jabref/gui/theme/ThemeManager.java

```diff
@@ -121,7 +121,11 @@ private void applyDarkModeToWindow(Stage stage, boolean darkMode) {
             return;
         }
 
-        themeWindowManager.setDarkModeForWindowFrame(stage, darkMode);
+        try {
+            themeWindowManager.setDarkModeForWindowFrame(stage, darkMode);
```

Comment:
Wrap the         this.themeWindowManager = ThemeWindowManagerFactory.create() also in an expcetion blog